### ISSUE TITLE
Content Security Policy error for the progress bar inline CSS

### ISSUE THREAD

**Issue by evgenyneu** [hotwired/turbo#781]
After enabling Content Security Policy in Rails I get the following error in Google Chrome from `turbo.es2017-esm.js:2059`:

> Refused to apply inline style because it violates the following Content Security Policy directive: "default-src 'self'". Either the 'unsafe-inline' keyword, a hash ('sha256-rql2tlBWA4Hb3HHbUfw797urk+ifBd6EAovoOUGt0oI='), or a nonce ('nonce-...') is required to enable inline execution. Note also that 'style-src' was not explicitly set, so 'default-src' is used as a fallback.

This is probably about this CSS being injected into the head:

```HTML
<style type="text/css">
.turbo-progress-bar {
  position: fixed;
  display: block;
  top: 0;
  left: 0;
  height: 3px;
  background: #0076ff;
  z-index: 9999;
  transition:
    width 300ms ease-out,
    opacity 150ms 150ms ease-in;
  transform: translate3d(0, 0, 0);
}
</style>
```

This probably needs to be fixed by adding the CSP `nonce` attribute to the `style` element, similar to the change in https://github.com/hotwired/turbo/pull/291.

## Workaround

This works but sucks since it will break if the progress bar CSS changes:

```
policy.style_src   :self, "'sha256-rql2tlBWA4Hb3HHbUfw797urk+ifBd6EAovoOUGt0oI='"
```

## My content security configuration

```Ruby
config.content_security_policy do |policy|
    policy.default_src :self
end

config.content_security_policy_nonce_generator = -> (request) { SecureRandom.base64(16) }
config.content_security_policy_nonce_directives = %w(script-src style-src)
```

## Versions

I'm using turbo-rails 1.3.2 in rails 7.0.3.1.

**Comment by marcoroth** [comment]
Hey @evgenyneu, thanks for opening this issue! This might be related to https://github.com/hotwired/turbo/issues/704.

Does the fix in https://github.com/hotwired/turbo/issues/704#issuecomment-1261090003 solve your problem?

**Comment by evgenyneu** [comment]
@marcoroth apologies for the duplicate. No, https://github.com/hotwired/turbo/issues/704#issuecomment-1261090003 does not solve my problem, in fact this is the CSP configuration I was already using. 

I looks like the bug was already fixed in [turbo v7.2.0](https://github.com/hotwired/turbo/releases/tag/v7.2.0) by adding the nonce to the progress bar `style` element. But the nonce is not present for me (as you can see above for the `.turbo-progress-bar` CSS. When I click on `turbo.es2017-esm.js:2059` from Google Chrome console it show that the Turbo JS code is version 7.1.0, so this fix is not present there. Why am I loading the old turbo js? I'm using the latest turbo-rails v1.3.2, which [should load turbo v7.2.4](https://github.com/hotwired/turbo-rails/releases/tag/v1.3.2)?
